From 45b2ef221da53c6bbf62ede2ea5d3d4889a79365 Mon Sep 17 00:00:00 2001
From: Gal Pressman <galp@mellanox.com>
Date: Sun, 24 Apr 2016 22:51:48 +0300
Subject: net/mlx5e: Rename VPort counters
Reply-To: joseph.salisbury@microsoft.com

VPort and software counters names are confusing and may be unclear, all
VPort counters now have a prefix of rx/tx_vport_*.

Signed-off-by: Gal Pressman <galp@mellanox.com>
Signed-off-by: Saeed Mahameed <saeedm@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 8075cb72382bf854a3a95f74ea4f9d19ebe29fd5)
Signed-off-by: Joseph Salisbury <joseph.salisbury@microsoft.com>
---
 .../ethernet/mellanox/mlx5/core/en_stats.h    | 35 ++++++++++---------
 1 file changed, 19 insertions(+), 16 deletions(-)

diff --git a/drivers/net/ethernet/mellanox/mlx5/core/en_stats.h b/drivers/net/ethernet/mellanox/mlx5/core/en_stats.h
index 116320d8fc42..4f3a08d7e8ed 100644
--- a/drivers/net/ethernet/mellanox/mlx5/core/en_stats.h
+++ b/drivers/net/ethernet/mellanox/mlx5/core/en_stats.h
@@ -115,32 +115,35 @@ struct mlx5e_vport_stats {
 };
 
 static const struct counter_desc vport_stats_desc[] = {
-	{ "rx_error_packets", VPORT_COUNTER_OFF(received_errors.packets) },
-	{ "rx_error_bytes", VPORT_COUNTER_OFF(received_errors.octets) },
-	{ "tx_error_packets", VPORT_COUNTER_OFF(transmit_errors.packets) },
-	{ "tx_error_bytes", VPORT_COUNTER_OFF(transmit_errors.octets) },
-	{ "rx_unicast_packets",
+	{ "rx_vport_error_packets",
+		VPORT_COUNTER_OFF(received_errors.packets) },
+	{ "rx_vport_error_bytes", VPORT_COUNTER_OFF(received_errors.octets) },
+	{ "tx_vport_error_packets",
+		VPORT_COUNTER_OFF(transmit_errors.packets) },
+	{ "tx_vport_error_bytes", VPORT_COUNTER_OFF(transmit_errors.octets) },
+	{ "rx_vport_unicast_packets",
 		VPORT_COUNTER_OFF(received_eth_unicast.packets) },
-	{ "rx_unicast_bytes", VPORT_COUNTER_OFF(received_eth_unicast.octets) },
-	{ "tx_unicast_packets",
+	{ "rx_vport_unicast_bytes",
+		VPORT_COUNTER_OFF(received_eth_unicast.octets) },
+	{ "tx_vport_unicast_packets",
 		VPORT_COUNTER_OFF(transmitted_eth_unicast.packets) },
-	{ "tx_unicast_bytes",
+	{ "tx_vport_unicast_bytes",
 		VPORT_COUNTER_OFF(transmitted_eth_unicast.octets) },
-	{ "rx_multicast_packets",
+	{ "rx_vport_multicast_packets",
 		VPORT_COUNTER_OFF(received_eth_multicast.packets) },
-	{ "rx_multicast_bytes",
+	{ "rx_vport_multicast_bytes",
 		VPORT_COUNTER_OFF(received_eth_multicast.octets) },
-	{ "tx_multicast_packets",
+	{ "tx_vport_multicast_packets",
 		VPORT_COUNTER_OFF(transmitted_eth_multicast.packets) },
-	{ "tx_multicast_bytes",
+	{ "tx_vport_multicast_bytes",
 		VPORT_COUNTER_OFF(transmitted_eth_multicast.octets) },
-	{ "rx_broadcast_packets",
+	{ "rx_vport_broadcast_packets",
 		VPORT_COUNTER_OFF(received_eth_broadcast.packets) },
-	{ "rx_broadcast_bytes",
+	{ "rx_vport_broadcast_bytes",
 		VPORT_COUNTER_OFF(received_eth_broadcast.octets) },
-	{ "tx_broadcast_packets",
+	{ "tx_vport_broadcast_packets",
 		VPORT_COUNTER_OFF(transmitted_eth_broadcast.packets) },
-	{ "tx_broadcast_bytes",
+	{ "tx_vport_broadcast_bytes",
 		VPORT_COUNTER_OFF(transmitted_eth_broadcast.octets) },
 };
 
-- 
2.17.1

